taxonomy/management/commands/refresh_job_skills.py

```diff
@@ -62,7 +62,7 @@ def _refresh_jobs(self, ranking_facet, nested_ranking_facet):
         client = EMSIJobsApiClient()
         try:
             skills = Skill.objects.all()
-            for chunked_skills in chunked_queryset(skills):
+            for chunked_skills in chunked_queryset(skills, 50):
```

Comment:
Can we add the param name as well for better readbility?
`for chunked_skills in chunked_queryset(skills, chunk_size=50):`